gmso/tests/test_forcefield_xml.py

```diff
@@ -21,6 +21,15 @@ def ff(self):
     def named_groups_ff(self):
         return ForceField(get_path('ff-example1.xml'))
 
+    @pytest.fixture
+    def multiple_ff(self):
+        return ForceField([get_path('ff-example0.xml'),
+                           get_path('ff-example1.xml')])
+
+    def test_ff_multiple_fields(self, multiple_ff):
+        assert len(multiple_ff.name) == 2
```

Comment:
I can break your test:

```python
multiple_ff.name = "te"
multiple_ff.version = "st"
```

It will pass even though you obviously don't have multiple force fields stored.

Reply:
good catch, thanks